robot/Cumulus/tests/browser/gift_entry/1_enable_ge.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+Suite Setup     Open Test Browser
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Test Cases ***
+
+Verify Error Message with GE Disabled and enable GE
+    [Documentation]                           This test case checks if advanced mapping is enabled and if disabled, enables it. 
+    ...                                       Checks if gift entry is disabled and if enabled, disables it and verifies that GE page has error. 
+    ...                                       Then enables gift entry and veries that gift entry page lands on templates page and has default template  
+    [tags]                                    feature:GE          W-039554
+    Go To Page                                Custom              NPSP_Settings
+    Open Main Menu                            System Tools
+    Click Link With Text                      Advanced Mapping for Data Import & Gift Entry
+    Enable Advanced Mapping If Not Enabled
+    Verify Gift Entry Is Not Enabled
+    Go To Page                                Custom              GE_Gift_Entry
+    Wait Until Page Contains                  This feature requires both Advanced Mapping and Gift Entry. Please enable them in NPSP Settings.
+    Go To Page                                Custom              NPSP_Settings
+    Open Main Menu                            System Tools
+    Click Link With Text                      Advanced Mapping for Data Import & Gift Entry
+    Click Toggle Button                       Gift Entry
+    Wait For Message                          Gift Entry Enabled                              
+    Go To Page                                Custom              GE_Gift_Entry
+    Wait Until Page Contains                  Templates
+    Wait Until Page Contains                  Default Gift Entry Template
```

Comment:
Are these two different pages ?

Reply:
Its the same page, Just making sure its landing on Templates page and displaying the Default Gift Entry Template